TraceScripts/README.md

```diff
@@ -92,4 +92,27 @@ $> adb push <filepath>/PlayFabMultiplayerLogger.json /sdcard/PlayFabLogs/config/
     Please note that if you've modified the `logFolder` property of your `PlayFabMultiplayerLogger.json`, then you'll need to update the ADB command accordingly.
 ```
 $> adb pull /sdcard/PlayFabLogs/log/ <target local directory>
-```
\ No newline at end of file
+```
+
+### Linux: Instructions
+
+1. Download the [`PlayFabMultiplayerLogger.json` file](Linux/PlayFabMultiplayerLogger.json) from this repo.
+2. Open a new terminal.
+3. Create your `config` directory on the Linux device.
+```
+$> mkdir $HOME/PlayFabLogs/config/
+```
+4. You will need to modify the `logFolder` property of the `PlayFabMultiplayerLogger.json` file.
+```
+"logFolder": "/home/<your-username>/PlayFabLogs/log/"
+```
+- Please replace `<your-username>` with your username on your Linux machine. If you are unsure of your username:
+```
+$> echo $HOME
+```
+5. Copy the `PlayFabMultiplayerLogger.json` file into your `$HOME/PlayFabLogs/config/` directory.
+```
+$> cp <filepath>/PlayFabMultiplayerLogger.json $HOME/PlayFabLogs/config/
+```
+6. Run the application.
+7. Run the application. If using the default configuration of `PlayFabMultiplayerLogger.json`, logs should automatically be written to `/home/<your-username>/PlayFabLogs/log/`. If not, logs will be written to the directory specified by the `logFolder` property of `PlayFabMultiplayerLogger.json`.
```

Comment:
'If not, logs will be written to the directory specified by the `logFolder` property of `PlayFabMultiplayerLogger.json`.'

Probaly, the above sentence without the 'if not' part is sufficient for step 7. In any case, the user will have to go manually and see what is the value of 'logFolder'.

Reply:
I think Nassos's last suggestion for this line (at least how I read it) was more intended to be simplified like this

```suggestion
6. Run the application. Logs will be written to the directory specified by the `logFolder` property of `PlayFabMultiplayerLogger.json`.
```

This is simpler since there's no condition about whether you did or didn't change the default configuration

Reply:
agreed, updated